brave-core/src/main/java/com/github/kristofa/brave/ServerTracer.java

```diff
@@ -47,7 +47,7 @@ public Builder state(ServerSpanState state) {
             return spanAndEndpoint(ServerSpanAndEndpoint.create(state));
         }
 
-        abstract Builder spanAndEndpoint(ServerSpanAndEndpoint spanAndEndpoint);
+        public abstract Builder spanAndEndpoint(ServerSpanAndEndpoint spanAndEndpoint);
```

Comment:
I don't think this needs to be public as it does not need to be set by a user of the api to build a proper `ServerTracer`? `ServerSpanAndEndpoint` is an abstraction on top of `ServerSpanState` which can be configured using the Builder.